ui/cypress/integration/common/accessDocumentation.js

```diff
@@ -0,0 +1,26 @@
+import { Then } from 'cypress-cucumber-preprocessor/steps';
+
+Then('I access the documentation from the navbar', () => {
+  const target_url = Cypress.env('target_url');
+
+  cy.visit(target_url, {
+    onBeforeLoad(win) {
+      // win is the the remote page's window object
+      // `cy.stub()` stubs the window.open method
+      cy.stub(win, 'open', url => {
+        // cannot make a HTTP request to the documentation url because cypress doesn't support the `SameSite`
```

Comment:
I don't really understand this issue, what's this `SameSite` problem?

Reply:
As we discussed, we are not supposed to nest the `cy.resquest` inside the `cy.visit`
I just check if the doc URL is correct. 
For the doc endpoint, we will need a pytest to cover it.